plonky2/src/plonk/circuit_data.rs

```diff
@@ -247,40 +247,40 @@ pub struct CommonCircuitData<
     C: GenericConfig<D, F = F>,
     const D: usize,
 > {
-    pub(crate) config: CircuitConfig,
+    pub config: CircuitConfig,
 
-    pub(crate) fri_params: FriParams,
+    pub fri_params: FriParams,
```

Comment:
For this struct, I see why `config` and `degree_bits` need to be public now but it seems like the other fields can stay private, no?